docs/pct/README.md

```diff
@@ -0,0 +1,291 @@
+# Puppet Content Templates
+
+* [Overview](#overview)
+* [Getting Started](#getting-started)
+  * [pct new](#pct-new)
+  * [Template Updates](#template-updates)
+  * [More Templates](#more-templates)
+* [Writing Your Own Templates](#writing-templates)
+  * [Dos and Don'ts](#dos-and-donts)
+  * [Structure](#structure)
+    * [pct-config.yml](#pct-configyml)
+  * [Templating Language](#templating-language)
+  * [Testing](#testing)
+  * [Sharing](#sharing)
+* [Overriding Template Defaults](#overriding-template-defaults)
+
+## Overview
+
+`Puppet Content Templates` (PCT) allow you to quickly create Puppet content from scaffolding templates built by Puppet and the Community.
+
+> :warning: PCT is currently in an EXPERIMENTAL phase and feedback is encouraged via [pdk/discussions](https://github.com/puppetlabs/pdk/discussions) and starting a `feedback` post.
+
+## Getting Started
+
+Install the `exprimental` PCT release available from [github.com/puppetlabs/pdkgo/releases/](https://github.com/puppetlabs/pdkgo/releases/)
```

Comment:
Would use the word 'download' here as we don't yet have an installer :)

Reply:
this may be on a do to list or covered elsewhere, but how do we set up the experimental PCT?

Reply:
> this may be on a do to list or covered elsewhere, but how do we set up the experimental PCT?

We don't have the PCT in this state. I've written some of this documentation based on where i think we need to be in order to release an LA and to stimulate discussion.